spec/predicates/runtime-trace.md

```diff
@@ -18,9 +18,12 @@ target process.
 
 ## Use Cases
 
-Generally, this predicate can be used with a runtime trace of any process.
-However, the primary use case driving the development of this predicate is its
-applicability to runtime traces of build processes.
+Generally, this predicate can be used with a runtime trace of any process. The
+attestation specification does not mandate any specific monitoring tool or
```

Comment:
Do you mean the "predicate specification" doesn't mandate specific monitoring tools? I would also disambiguate the use of the term "process" wherever possible (in some places you probably mean "operation" rather an actual OS process).

Reply:
Should `monitoredProcess` be renamed?

Reply:
Updated the text generally to use "operation" / "step" in several places instead of "process"